impl/src/main/resources/META-INF/permittedTaglibs.tld

```diff
@@ -22,29 +23,28 @@
     xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-jsptaglibrary_2_0.xsd"
     version="2.0">
   <description>
-    Restricts JSP pages to the JSTL tag libraries
+    Restricts JSP pages to the Tags libraries
   </description>    
   <display-name>permittedTaglibs</display-name>
-  <tlib-version>1.1</tlib-version>
+  <tlib-version>3.0</tlib-version>
   <short-name>permittedTaglibs</short-name>
-  <uri>http://jakarta.apache.org/taglibs/standard/permittedTaglibs</uri>
+  <uri>jakarta.tags.permittedTaglibs</uri>
 
   <validator>
     <validator-class>
 	jakarta.servlet.jsp.jstl.tlv.PermittedTaglibsTLV
     </validator-class>        
     <init-param>
       <description>
-        Whitespace-separated list of taglib URIs to permit.  This example
-	TLD for the Standard Taglib allows only JSTL 'el' taglibs to be
-	imported.
+        Whitespace-separated list of taglib URIs to permit. 
+        This list only permits the EE10 jakarta.tags URIs.
```

Comment:
I'd like to preserve more of this description.

Whitespace-separated list of taglib URIs to permit. This example TLD for the Standard Taglib allows only Jakarta Standard Tab Library `el` taglibs to be imported. This list only permit the EE10 jakarta.tags URIs.